assets/quick-buy.css

```diff
@@ -0,0 +1,197 @@
+.quick-buy-modal {
+  box-sizing: border-box;
+  opacity: 0;
+  position: fixed;
+  visibility: hidden;
+  z-index: -1;
+  margin: 0 auto;
+  top: 0;
+  left: 0;
+  overflow: auto;
+  width: 100%;
+  background: rgba(var(--color-foreground), 0.2);
+  height: 100%;
+}
+
+.quick-buy-modal[open] {
+  opacity: 1;
+  visibility: visible;
+  z-index: 101;
+}
+
+.quick-buy-modal__content {
+  border-radius: var(--popup-corner-radius);
+  background-color: rgb(var(--color-background));
+  overflow: auto;
+  margin: 0 auto;
+  left: 50%;
+  transform: translateX(-50%);
+  margin-top: 3.2rem;
+  width: 92%;
+  position: absolute;
+  top: 0;
+  padding: 0 1.5rem 0 3rem;
+  border-color: rgba(var(--color-foreground), var(--popup-border-opacity));
+  border-style: solid;
+  border-width: var(--popup-border-width);
```

Comment:
Could this be maybe grouped? Or just easier like this for readability ? I don't have strong feeling about it. Im ok if you leave it as is.

Reply:
I thought about grouping it too, but it's being used in many places in this exact same way for readability purposes:

<img width="365" alt="image" src="https://user-images.githubusercontent.com/37168033/158102619-c2d043ec-a1b3-47d4-af92-2342b293413a.png">